docs/content/deploying-to-hippo.md

```diff
@@ -30,72 +32,57 @@ coming soon.
-
-```
-$ git clone git@github.com:deislabs/hippo.git
-$ cd hippo
-$ export BINDLE_URL=http://bindle.local.fermyon.link:8088/v1
-$ dotnet clean && dotnet build
-$ cd src/Web
-$ dotnet run \
-  --Scheduler:Driver=nomad \
-  --Bindle:Url="${BINDLE_URL}"
-```
-
-Check that the Hippo platform is running by making sure you see the hippo dashboard in the browser at https://localhost:5309.
+Check that the Hippo platform is running by making sure you see the hippo
+dashboard in the browser at <http://hippo.local.fermyon.link>
 
 Register an account in Hippo using the Hippo CLI.
 
 ```
-hippo auth register --username user --password PassW0rd! --url https://localhost:5309 -k
+$ hippo register --username user --password PassW0rd! --url http://hippo.local.fermyon.link
 ```
 
 ### Deploy the example http-rust app
 
-Clone the [Spin repo](https://github.com/fermyon/spin) to ensure you have the examples locally. Make sure that the http-rust example is built before deploying.
+Clone the [Spin repo](https://github.com/fermyon/spin) to ensure you have the
+examples locally. Make sure that the http-rust example is built before
+deploying.
 
 ```
 $ git clone git@github.com:fermyon/spin.git
 $ cd spin/examples/http-rust
 $ spin build
 ```
 
-Then, set the relevant environment variables and use `spin deploy` to deploy the app to Hippo. _Note: you can also use flags to configure spin deploy_
+Then, set the relevant environment variables and use `spin deploy` to deploy
+the app to Hippo. _Note: you can also use flags to configure spin deploy_
 
 ```
-$ export HIPPO_URL=https://localhost:5309
-$ export BINDLE_URL=http://bindle.local.fermyon.link:8088/v1
+$ export HIPPO_URL=http://hippo.local.fermyon.link
+$ export BINDLE_URL=http://bindle.local.fermyon.link/v1
 $ export HIPPO_USERNAME=user
 $ export HIPPO_PASSWORD=PassW0rd!
-
-$ spin deploy -k
-
+$ spin deploy
 ```
 
-The spin deploy command packages the application and pushes it to the bindle registry, creates a new app in Hippo and a new channel that the runs the application.
+The `spin deploy` command packages the application and pushes it to the bindle
+registry, creates a new app in Hippo and a new channel that the runs the
+application.
 
 ### Test the app
 
-In the future, spin deploy will give the user a domain to hit for the running app. In the meantime, find the IP address and port for the running app int he Nomad dashboard.
-
-Check the Nomad UI for the running job.
+In the future, `spin deploy` will give the user a domain to hit for the running
+app.
 
 ```
-# this command opens up the nomad dashboard in your browser
-$ nomad ui
+$ curl http://spin-deploy.spin-hello-world.hippo.local.fermyon.link/hello
```

Comment:
Note that this URL is currently incorrect. At the moment Spin creates a channel called `spin-hello-world` resulting in the domain http://spin-hello-world.spin-hello-world.hippo.local.fermyon.link/hello

IIRC there is work in-development to change this, so this should be changed to reflect the new URL once complete.

cc @michelleN

Reply:
Also, with `spin 0.2.0 (60907c4 2022-06-10)` I'm seeing an app URL with no `hippo` section:
<img width="408" alt="Screen Shot 2022-06-13 at 10 26 28 AM" src="https://user-images.githubusercontent.com/9322017/173400470-1b9aef84-0433-439a-9b77-8c6240fcafd0.png">

Reply:
Good catch. I believe this is because [nomad-local-demo uses `local.fermyon.link`](https://github.com/fermyon/nomad-local-demo/blob/2082ef381493bd3438df5f35921d06bbad87d9ac/job/hippo-linux.nomad#L50) as the platform domain instead of `hippo.local.fermyon.link`. [nomad-aws-demo uses `hippo.ip_address.sslip.io`](https://github.com/fermyon/installer/blob/bbcea30bb7a80f9b71ae62c07fd3abf8a63e5a1a/aws/quick-start/terraform/ec2_assets/run_servers.sh#L97) as the platform domain.

Both work... I just preferred to use the `hippo.domain` convention to signify to the user that "this app is running on Hippo".

We have two options:

1. update nomad-local-demo to match nomad-aws-demo (no changes to this doc required)
2. update nomad-aws-demo to match nomad-local-demo (changes to this doc required)

Let's chat about this offline

cc @adamreese

Reply:
Ah, thanks for comparing scenarios.  Indeed, we should decide on a consistent approach. 👍

Reply:
the channel name has been updated to be spin-deploy!